lesson27/js/token.js

```diff
@@ -0,0 +1,3 @@
+localStorage.getItem("token")
+  ? (window.location.href = "./news.html")
+  : (window.location.href = "./login.html");
```

Comment:
IMO
It will be limited.
`(window.location.href = "./news.html")`
I feel like I don't need this!

Reply:
Thank you!!
If I implement like below, it will stop working when I open index.html if it has token.
`if (!localStorage.getItem("token")) window.location.href = "./login.html";`

I think I might misunderstand...
Is this `index.html` mentioned in this lesson perhaps referring to the news content?
I had created a new index.html separate from the news content html💦

Reply:
Yes, the contents of index.html should contain newscontent!
[codesandbox](https://codesandbox.io/s/lesson27-forked-e1irqw?file=/login.html)
This way, even if you intentionally delete the local storage after logging in, you won't be redirected to the content page.

Reply:
I appreciate your kind instruction. I've fixed about this.
[633274e](https://github.com/yuka830/js-lessons/pull/32/commits/633274e4288e53c83d84a234e788959dadda707e)

> [9739583](https://github.com/yuka830/js-lessons/pull/32/commits/9739583076248e484301544daf9cb8c2c73ace62) ←This part has been removed from this sandbox.

Oh, I think I might change something when you were checking my CodeSandbox.
Sorry about that.